aqueduct_client/aqueduct_client.py

```diff
@@ -105,6 +108,28 @@ def get_pipeline_execution(self, execution_id):
         pipeline = response.json()['pipeline']
         return pipeline
 
+    def get_active_pipeline_executions_info(self):
+        """
+        Returns the number of currently active (queued or running)
+        pipeline executions, and what the quota is.
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        dict: A dictionary with the following keys:
+            concurrent_executions_count: int
```

Comment:
These seem like pretty verbose names. Would we shorten these to something like `running` and `max_running` or `quota`?

I also wonder if we'll eventually want to expand this to provide info on other kinds of API limits (e.g., concurrent backtest executions). I could imagine having a more general "quotas" or "limits" concept with a structure like:

```
def get_quotas(self, resources=None):
    """
    Get quota status for one or more resources.

    Parameters
    ----------
    resources : list[str], optional
        List of resources for which to fetch quotas. By default, all quotas are
        retrieved. Possible options include 'pipeline_executions', 'backtests',
        etc.

    Returns
    -------
    quotas : list
        List of quotas for requested resources. Quotas are returns as
        dictionary of the form::

            {
                'resource': 'pipeline_executions',
                'running': 2,
                'maximum': 5,
            }
    """
    response = self._get('/quotas', params={'resources': resources})
    # ...

```

Reply:
Agreed on both.  Shortening the keys now (to `running` and `maximum`), will hold off on generalizing the quota system for now but it seems likely to happen.